### ISSUE TITLE
Unable to see radio button selection with light theme

### ISSUE THREAD

**Issue by gquittet** [catppuccin/jetbrains#77]
Unable to see radio button selection with light theme: Catppuccin Latte

It works great with dark theme: Catppuccin Mocha

#### Light Theme

![image](https://github.com/catppuccin/jetbrains/assets/11082739/db8132ec-c75d-457b-9cea-4b84e2403a42)

#### Dark Theme

![image](https://github.com/catppuccin/jetbrains/assets/11082739/0b0c7715-c129-4ed8-bcf9-22242e9318ec)

---

```txt
IntelliJ IDEA 2023.1.2 (Ultimate Edition)
Build #IU-231.9011.34, built on May 16, 2023
New Beta UI enabled 
```

**Comment by gquittet** [comment]
I just tested the new update and the issue is still there.

<img width="444" alt="image" src="https://github.com/catppuccin/jetbrains/assets/11082739/b93bf07b-ad27-429d-8114-7c88c3edeb26">

**Comment by sgoudham** [comment]
Hi, I finally had some time to dig into this. 

## TLDR

JetBrains stopped theming the radio buttons in light mode resulting in the icon and the background being the same colour

## The Problem
Radio Buttons are _really weird_ in JetBrains, mostly because they seem to inherit the colours for `CheckBox` instead of the actual `RadioButton`

<details>
<summary>See Evidence Below of the RadioButton keys not being respected</summary>

`RadioButton Keys`
![image](https://github.com/catppuccin/jetbrains/assets/58985301/b12f57fa-c300-4c31-a5ff-1bdd03ad03cf)

`The Result`
![image](https://github.com/catppuccin/jetbrains/assets/58985301/815be449-21ab-42eb-a680-6677f2e3e1ea)

</details>

## The Fix/Workaround

Unfortunately, the only way I can think of to make this usable is to make the default colour for the checkbox darker (as the radio button will inherit that) Here's an example of what the fix/workaround would look like:

![image](https://github.com/catppuccin/jetbrains/assets/58985301/bac395a9-5bbe-40cf-8576-ba1f7c28d21d)

I can't think of any other way around this until JetBrains fix this, what do you think? @gquittet

**Comment by gquittet** [comment]
Thanks for the crystal clear explanation 🙏
It's a great solution!
